filesystem/squashfs_superblock.ksy

```diff
@@ -0,0 +1,116 @@
+meta:
+  id: squashfs_superblock
+  title: SquashFS superblock
+  file-extension:
+    - squashfs
+    - snap
+  endian: le
```

Comment:
Actually SquashFS can also be big endian:

https://github.com/plougher/squashfs-tools/blob/master/squashfs-tools/squashfs_fs.h#L30

Reply:
BTW KS has switcheable endianness.

Reply:
SquashFS 4.0 is *strictly* little endian. The older versions of SquashFS *could* be either.

This definition is only there for compatibility, since `unsquashfs` supports unpacking older formats.

There are some big endian 4.0 images floating around, typically created by cheap plastic router vendors who thought it's a great idea to hack up the format.